microprofile/server/src/main/java/io/helidon/microprofile/server/Server.java

```diff
@@ -107,7 +94,9 @@ static Builder builder() {
      * Get CDI container in use.
      *
      * @return CDI container instance (standard edition)
+     * @deprecated use {@link javax.enterprise.inject.spi.CDI#current()}
      */
+    @Deprecated
```

Comment:
Maybe therefore provide a `default` implementation here?

Reply:
I have removed the method.